README.md

```diff
@@ -53,15 +53,18 @@ Usage
 Authors
 -------
 
-### Maintainers and core developers
+### Maintainers* and core developers
```

Comment:
Unrelated to this PR, but in this header, only the first word starts with a capital letter, while in others each work is. Btw in PIConGPU it is the same inconsistency with this header.